frontend/app_flowy/assets/translations/en.json

```
@@ -163,7 +164,29 @@
         "light": "Light Mode",
         "dark": "Dark Mode",
         "system": "Adapt to System"
-      }
+      },
+      "lightLabel": "Light Mode",
+      "darkLabel": "Dark Mode"
```

```suggestion
      }
```

We already have this a few lines above.

---

My fault. I mishandle this conflict here during rebasing.